spec/parsers/aac_parser_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+
+describe FormatParser::AACParser do
+  it 'should match filenames with valid AAC extensions' do
+    filenames = ['audiofile', 'audio_file', 'audio-file', 'audio file', 'audio.file']
+    extensions = ['.aac', '.AAC', '.Aac', '.AAc', '.aAc', '.aAC', '.aaC']
+    filenames.each { |filename|
```

Comment:
`do...end` is preferred for multiline blocks 👍

Reply:
Done.